### PR TITLE
Only use Math.imul/clz32 when available

### LINKED ISSUE #47: Possibly broken in IE11
I noticed that JSBI is broken in a test application when running in IE11.

When I exclude JSBI in webpacks babel-loader exclude (jsbi is parsed like the rest of my code):

grafik
This possibly points to:
```javascript
if (!_(t) || i(t) !== t) throw new RangeError("The number " + t + " cannot be converted to BigInt because it is not an integer");
```
I use core-js@3 and regenerator-runtime as polyfills. Maybe I need something else as well?

When I do not exlude JSBI and treat it like any other dependency (jsbi is included in babel-loader):
grafik
This time it is unfortunately absolutely impossible to tell what upsets poor IE. Maybe its the same statement, who knows 🤷‍♂️.

Is the UMD version tested? Works fine in Firefox 80, but not in IE11.

I also noticed https://github.com/GoogleChromeLabs/jsbi/issues/42. I tested importing the common-js and mjs versions directly, however I still get similar results. IE is broken while Firefox works.

### PR DESCRIPTION
**Pull request by jakobkummerow**
Fall back to custom implementations otherwise.
This is for IE11 compatibility.

Fixes #47.

### CODE REVIEW

File: jsbi.mjs [hunk 1]
```diff
@@ -1905,4 +1905,13 @@ JSBI.__kBitConversionBuffer = new ArrayBuffer(8);
 JSBI.__kBitConversionDouble = new Float64Array(JSBI.__kBitConversionBuffer);
 JSBI.__kBitConversionInts = new Int32Array(JSBI.__kBitConversionBuffer);
 
+// For IE11 compatibility.
```
Comment (mathiasbynens) [inline jsbi.mjs:1908]: Do we want to mention in a comment that the fallback helper functions are simplified versions, and thus not re-usable as polyfills?

**Review by mathiasbynens — APPROVED** [review]
LGTM with a suggestion

File: jsbi.mjs (hunk 1 above)
Comment (jakobkummerow) [reply jsbi.mjs:1908]: Done.

**Comment by mathiasbynens** [comment]
Capturing the offline discussion: contrary to [common usage](https://github.com/sindresorhus/ponyfill#how-are-ponyfills-better-than-polyfills), we decided to use the built-ins when available before falling back to the custom implementations because they pose a performance benefit.

**Pull request merged**